source/material_model/reaction_model/grain_size_evolution.cc

```diff
@@ -417,6 +416,18 @@ namespace aspect
         prm.declare_entry ("Advect logarithm of grain size", "false",
                            Patterns::Bool (),
                            "This option does not exist any more.");
+        prm.declare_entry ("Initial step size", "1e-3",
+                           Patterns::Double (0., 1.0),
```

Comment:
What happens if someone uses zero? Did you make sure they get a reasonable error message?